src/gatsby/models/blog/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  createPages: require('./createPages.js'),
+  onCreateMarkdownContentNode: require('./onCreateNode.js'),
+  createSchemaCustomization: require('./createSchemaCustomization')
```

Comment:
should it end with `.js` like everywhere else?

Reply:
Good catch! seems like neither linters nor I picked this up.

Reply:
agreed - strange that linters allow both ways